torch/csrc/autograd/python_variable.cpp

```diff
@@ -163,7 +163,7 @@ PyObject *THPVariable_get_cdata(THPVariable *self)
 {
   HANDLE_TH_ERRORS
   auto& var = self->cdata;
-  return PyLong_FromVoidPtr(var.data().unsafeGetTensorImpl());
+  return PyLong_FromVoidPtr(var.unsafeGetTensorImpl());
```

Comment:
`THPVariable_get_cdata` is meant to return the address of the data tensor's TensorImpl. Since now that the Variable is the data tensor itself, it's okay to remove the `.data()` call here.